app/modules/elasticsearch/tasks.py

```diff
@@ -128,7 +128,43 @@ def load_encounters_index():
 """
 
 
+def load_sightings_index():
+    wb_engine = create_wildbook_engine()
+    with wb_engine.connect() as wb_conn:
+        results = wb_conn.execute(text(SIGHTINGS_INDEX_SQL))
+        for result in results:
+            # Create the document object
+            sighting = Sighting(**result)
+            # Assign the elasticsearch document identify
+            sighting.meta.id = f'sighting_{sighting.id}'
+            # Save document to elasticsearch
+            sighting.save(using=current_app.elasticsearch)
+
+
+SIGHTINGS_INDEX_SQL = """\
+SELECT
+  oc."ID" AS id,
+  NULLIF((oc."DECIMALLATITUDE"::float || ',' || oc."DECIMALLONGITUDE")::text, ',') AS point,
+  (
+    SELECT max("GENUS")
```

Comment:
`GENUS` and `SPECIES` should be ignored (again, migration only).  instead look to `TAXONOMY_ID_OID` joining to the `TAXONOMY` table -- this is where species related info now lives in edm.

(the latest zebra test db i have now _has this populated_ -- sorry it was missing from what you currently have.)